# PositionDebug.msg
Header header
geometry_msgs/PointStamped position